docs/user-guide/kubeadm-reference.md

```diff
@@ -0,0 +1,144 @@
+# kubeadm reference
+
+This document specifies the advanced options of kubeadm.
+
+When run, `kubeadm init` will bootstrap a Kubernetes cluster. This is made up
```

Comment:
"Running `kubeadm init` bootstraps a Kubernetes cluster. This consists of the following steps:"